assets/script.js

```diff
@@ -128,10 +129,43 @@ function displayRecipes(url) {
         let buttonEl = $("<button>")
         .addClass("btn btn-secondary modal-btns")
         // add attribute as an identifier for modal button to use to get recipie info from API
-        .attr("id", mealID)
+        .attr("id", recipeURL)
         .text("Cooking instructions");
 
-      recipeDiv.append(headerEl, recipeEl, buttonEl);
+      // Favourite button div
+      let favouriteDiv = $("<div>")
+      
+        // Favourite button created and added to card here
+        let buttonFavourite = $("<button>")
+        .addClass("btn btn-danger favourite-btns")
+        // add attribute as an identifier for favourite button
+        .attr("id", mealID)
+        .text("Favourite");
+        // Favourite icon created and added here
+        let iOne =$("<i>")
```

Comment:
Should we remove this icon if we're not using it?

Reply:
This icon is actually in use Naomi, the black outline of a heart is this icon, and the second icon (iTwo) actually overlays on top of it. I'm not sure if it's the best way of doing it but it seems to work atm, if you are happy to leave it like that for the time being I can modify the approach later if it becomes an issue.

Reply:
Ok sure - thanks for clarifying Chris.